### ISSUE TITLE
On nvidia devices, wezterm keeps sleeping after system suspend

### ISSUE THREAD

**Issue by rwxe** [wez/wezterm#4404]
### What Operating System(s) are you seeing this problem on?

Linux X11

### Which Wayland compositor or X11 Window manager(s) are you using?

```sh
echo $XDG_CURRENT_DESKTOP
pop:GNOME
```

### WezTerm version

wezterm 20231001-114302-385c8745

### Did you try the latest nightly build to see if the issue is better (or worse!) than your current version?

Yes, and I updated the version box above to show the version of the nightly that I tried

### Describe the bug

I have a Linux laptop running PopOS 20.10, which comes with its own proprietary Nvidia drivers. Once, after I closed the  lid and reopened it, I found that WezTerm was continuously freezing. I realized this was caused by the system suspend. Upon resuming from suspend, WezTerm kept trying to read events from `/dev/nvidia0`. However, for some unknown reason, it was unable to read from it, causing WezTerm to hang indefinitely. I had to kill the process by running `pkill wezterm-gui` from another terminal and then restart WezTerm. The system log and info I will post at the end. 

Now I avoid this problem by disabling closing the lid to suspend the system. Whatever this is a wezterm bug or a bug in my system, thanks in advance.

### To Reproduce

1. Start wezterm
2. Suspend the system via the GUI or closing the lid
3. Log in to the system again
4. wezterm freezes and keep sleeping

### Configuration

```lua
local wezterm = require 'wezterm'
local act = wezterm.action
local config = {}
if wezterm.config_builder then
	config = wezterm.config_builder()
end
config.font_size = 13.0
config.font = wezterm.font_with_fallback({
	"Fira Code",
	"Noto Sans CJK SC",
})
config.freetype_load_target = 'Normal'
config.freetype_render_target = 'Normal'
config.use_ime = true
config.keys = {
	{
		key = 'Enter',
		mods = 'ALT',
		action = act.DisableDefaultAssignment,
	},
	{
		key = 'F11',
		mods = '',
		action = act.ToggleFullScreen,
	},
	{
		key = '-',
		mods = 'CTRL|ALT|WIN',
		action = act.SplitVertical { domain = 'CurrentPaneDomain' },
	},
	{
		key = '=',
		mods = 'CTRL|ALT|WIN',
		action = act.SplitHorizontal { domain = "CurrentPaneDomain" },
	},
	{
		key = 'LeftArrow',
		mods = 'CTRL|WIN',
		action = act.ActivatePaneDirection 'Left',
	},
	{
		key = 'h',
		mods = 'CTRL|WIN',
		action = act.ActivatePaneDirection 'Left',
	},
	{
		key = 'RightArrow',
		mods = 'CTRL|WIN',
		action = act.ActivatePaneDirection 'Right',
	},
	{
		key = 'l',
		mods = 'CTRL|WIN',
		action = act.ActivatePaneDirection 'Right',
	},
	{
		key = 'UpArrow',
		mods = 'CTRL|WIN',
		action = act.ActivatePaneDirection 'Up',
	},
	{
		key = 'k',
		mods = 'CTRL|WIN',
		action = act.ActivatePaneDirection 'Up',
	},
	{
		key = 'DownArrow',
		mods = 'CTRL|WIN',
		action = act.ActivatePaneDirection 'Down',
	},
	{
		key = 'j',
		mods = 'CTRL|WIN',
		action = act.ActivatePaneDirection 'Down',
	},
	{
		key = 'LeftArrow',
		mods = 'CTRL|SHIFT|WIN',
		action = act.AdjustPaneSize { "Left", 1 },
	},
	{
		key = 'RightArrow',
		mods = 'CTRL|SHIFT|WIN',
		action = act.AdjustPaneSize { "Right", 1 },
	},
	{
		key = 'UpArrow',
		mods = 'CTRL|SHIFT|WIN',
		action = act.AdjustPaneSize { "Up", 1 },
	},
	{
		key = 'DownArrow',
		mods = 'CTRL|SHIFT|WIN',
		action = act.AdjustPaneSize { "Down", 1 },
	},
	{
		key = 'z',
		mods = 'CTRL|WIN',
		action = wezterm.action.TogglePaneZoomState,
	},
}
for i = 1, 8 do
	table.insert(config.keys, {
		key = tostring(i),
		mods = 'ALT',
		action = act.ActivateTab(i - 1),
	})
end
config.mouse_bindings = {
	{
		event = { Up = { streak = 1, button = 'Left' } },
		mods = 'NONE',
		action = act.CompleteSelection 'ClipboardAndPrimarySelection',
	},
	{
		event = { Up = { streak = 1, button = 'Left' } },
		mods = 'CTRL',
		action = act.OpenLinkAtMouseCursor,
	},
}
config.color_scheme = 'iTerm2 Default'
config.default_cursor_style = "BlinkingBlock"
config.cursor_blink_rate = 600
config.cursor_blink_ease_in = 'Constant'
config.cursor_blink_ease_out = 'Constant'
config.window_background_opacity = 0.85
config.enable_tab_bar = true
config.enable_scroll_bar = true
config.inactive_pane_hsb = {
	saturation = 0.8,
	brightness = 0.7,
}
config.window_padding = {
	left = 0,
	right = 0,
	top = 0,
	bottom = 0,
}
config.window_frame = {
	font = wezterm.font { family = 'Triplicate A Code', weight = 'Bold' },
	font_size = 10.0,
}
config.colors = {
	tab_bar = {
		active_tab = {
			bg_color = '#81d8d0',
			fg_color = 'Black',
		},
		inactive_tab = {
			bg_color = '#333333',
			fg_color = 'Silver',
		},
	},
	foreground = 'white',
	background = 'black',
	cursor_bg = '#10B9C4',
	cursor_fg = 'black',
	cursor_border = '#f37021',
	selection_fg = 'black',
	selection_bg = '#fffacd',
	scrollbar_thumb = '#222222',
	split = '#81d8d0',
	compose_cursor = 'orange',
	copy_mode_active_highlight_bg = { Color = '#000000' },
	copy_mode_active_highlight_fg = { AnsiColor = 'Black' },
	copy_mode_inactive_highlight_bg = { Color = '#52ad70' },
	copy_mode_inactive_highlight_fg = { AnsiColor = 'White' },
	quick_select_label_bg = { Color = 'peru' },
	quick_select_label_fg = { Color = '#ffffff' },
	quick_select_match_bg = { AnsiColor = 'Navy' },
	quick_select_match_fg = { Color = '#ffffff' },
}
return config
```

### Expected Behavior

wezterm don`t keep sleeping

### Logs

6133 is the sleeping wezterm process. the Chinese below `被请求设置未找到` means `The requested setting was not found`.
```
cat wezterm-gui-log-6133.txt 
16:29:59.540  WARN   window::os::x11::connection > Unable to resolve appearance using xdg-desktop-portal: get_appearance.read_setting: Reading xdg-portal org.freedesktop.appearance color-scheme: org.freedesktop.portal.Error.NotFound: 被请求设置未找到
```

### Anything else?

# Process Info

root@popos /var/log
\> # ps -ef |grep wezterm
```
username        6133    2227  0 16:29 ?        00:00:02 /usr/bin/wezterm-gui start --cwd .
```
root@popos /var/log
\> # cat /proc/6133/status
```
Name:	wezterm-gui
Umask:	0077
State:	S (sleeping)
Tgid:	6133
Ngid:	0
Pid:	6133
PPid:	2227
TracerPid:	0
Uid:	1000	1000	1000	1000
Gid:	1000	1000	1000	1000
FDSize:	64
Groups:	4 27 1000
NStgid:	6133
NSpid:	6133
NSpgid:	2227
NSsid:	2227
VmPeak:	 2684104 kB
VmSize:	 2671568 kB
VmLck:	       0 kB
VmPin:	       0 kB
VmHWM:	  171932 kB
VmRSS:	  171932 kB
RssAnon:	   54736 kB
RssFile:	  117124 kB
RssShmem:	      72 kB
VmData:	  309060 kB
VmStk:	     212 kB
VmExe:	   33392 kB
VmLib:	  169896 kB
VmPTE:	    1020 kB
VmSwap:	       0 kB
HugetlbPages:	       0 kB
CoreDumping:	0
THP_enabled:	1
Threads:	39
SigQ:	0/30916
SigPnd:	0000000000000000
ShdPnd:	0000000000000000
SigBlk:	0000000000000000
SigIgn:	0000000001001000
SigCgt:	0000000180000440
CapInh:	0000000000000000
CapPrm:	0000000000000000
CapEff:	0000000000000000
CapBnd:	000001ffffffffff
CapAmb:	0000000000000000
NoNewPrivs:	0
Seccomp:	0
Seccomp_filters:	0
Speculation_Store_Bypass:	thread vulnerable
SpeculationIndirectBranch:	conditional enabled
Cpus_allowed:	f
Cpus_allowed_list:	0-3
Mems_allowed:	00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000000,00000001
Mems_allowed_list:	0
voluntary_ctxt_switches:	58638
nonvoluntary_ctxt_switches:	211
```
root@popos /var/log
\> # strace -p 6133 # There will be unlimited output here
```
strace: Process 6133 attached
restart_syscall(<... resuming interrupted read ...
\>) = 0
poll([{fd=41, events=POLLIN|POLLPRI}], 1, 10) = 0 (Timeout)
poll([{fd=41, events=POLLIN|POLLPRI}], 1, 10) = 0 (Timeout)
poll([{fd=41, events=POLLIN|POLLPRI}], 1, 10^Cstrace: Process 6133 detached
 <detached ...
\>
```
root@popos /var/log
\> # ll /proc/6133/fd/41
```
lrwx------ 1 username username 64 10月  7 16:42 /proc/6133/fd/41 -
\> /dev/nvidia0
```
root@popos /var/log
\> # nvidia-smi
```
Sat Oct  7 16:42:11 2023
+-----------------------------------------------------------------------------+
| NVIDIA-SMI 470.57.02    Driver Version: 470.57.02    CUDA Version: 11.4     |
|-------------------------------+----------------------+----------------------+
| GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
|                               |                      |               MIG M. |
|===============================+======================+======================|
|   0  NVIDIA GeForce ...  Off  | 00000000:01:00.0 Off |                  N/A |
| N/A   47C    P0    N/A /  N/A |    486MiB /  2002MiB |      3%      Default |
|                               |                      |                  N/A |
+-------------------------------+----------------------+----------------------+
+-----------------------------------------------------------------------------+
| Processes:                                                                  |
|  GPU   GI   CI        PID   Type   Process name                  GPU Memory |
|        ID   ID                                                   Usage      |
|=============================================================================|
|    0   N/A  N/A      1631      G   /usr/lib/xorg/Xorg                200MiB |
|    0   N/A  N/A      2227      G   /usr/bin/gnome-shell              114MiB |
|    0   N/A  N/A      6133    C+G   /usr/bin/wezterm-gui               24MiB |
|    0   N/A  N/A      6948      G   /usr/lib/firefox/firefox          144MiB |
|    0   N/A  N/A      7316      G   /usr/lib/firefox/firefox            0MiB |
+-----------------------------------------------------------------------------+
```

# Syslog
```messages
Oct  7 16:30:07 popos NetworkManager[743]: <info>  [1696667407.4314] manager: sleep: sleep requested (sleeping: no  enabled: yes)
Oct  7 16:30:07 popos ModemManager[892]: <info>  [sleep-monitor] system is about to suspend
Oct  7 16:30:07 popos NetworkManager[743]: <info>  [1696667407.4315] device (enp3s0f1): state change: unavailable -> unmanaged (reason 'sleeping', sys-iface-state: 'managed')
Oct  7 16:30:07 popos NetworkManager[743]: <info>  [1696667407.4419] device (p2p-dev-wlp2s0): state change: disconnected -> unmanaged (reason 'sleeping', sys-iface-state: 'managed')
Oct  7 16:30:07 popos NetworkManager[743]: <info>  [1696667407.4425] manager: NetworkManager state is now ASLEEP
Oct  7 16:30:07 popos NetworkManager[743]: <info>  [1696667407.4430] device (wlp2s0): state change: activated -> deactivating (reason 'sleeping', sys-iface-state: 'managed')
Oct  7 16:30:07 popos dbus-daemon[740]: [system] Activating via systemd: service name='org.freedesktop.nm_dispatcher' unit='dbus-org.freedesktop.nm-dispatcher.service' requested by ':1.13' (uid=0 pid=743 comm="/usr/sbin/NetworkManager --no-daemon " label="unconfined")
Oct  7 16:30:07 popos systemd[1]: Starting Network Manager Script Dispatcher Service...
Oct  7 16:30:07 popos dbus-daemon[740]: [system] Successfully activated service 'org.freedesktop.nm_dispatcher'
Oct  7 16:30:07 popos systemd[1]: Started Network Manager Script Dispatcher Service.
Oct  7 16:30:07 popos kernel: [  801.700480] wlp2s0: deauthenticating from c2:24:80:7f:9d:a6 by local choice (Reason: 3=DEAUTH_LEAVING)
Oct  7 16:30:07 popos wpa_supplicant[784]: wlp2s0: CTRL-EVENT-DISCONNECTED bssid=c2:24:80:7f:9d:a6 reason=3 locally_generated=1
Oct  7 16:30:07 popos wpa_supplicant[784]: wlp2s0: CTRL-EVENT-SIGNAL-CHANGE above=0 signal=0 noise=9999 txrate=0
Oct  7 16:30:07 popos NetworkManager[743]: <info>  [1696667407.4732] device (wlp2s0): state change: deactivating -> disconnected (reason 'sleeping', sys-iface-state: 'managed')
Oct  7 16:30:07 popos avahi-daemon[735]: Withdrawing address record for fe80::9139:382a:f7a5:ed88 on wlp2s0.
Oct  7 16:30:07 popos avahi-daemon[735]: Leaving mDNS multicast group on interface wlp2s0.IPv6 with address fe80::9139:382a:f7a5:ed88.
Oct  7 16:30:07 popos avahi-daemon[735]: Interface wlp2s0.IPv6 no longer relevant for mDNS.
Oct  7 16:30:07 popos NetworkManager[743]: <info>  [1696667407.4967] dhcp4 (wlp2s0): canceled DHCP transaction
Oct  7 16:30:07 popos NetworkManager[743]: <info>  [1696667407.4967] dhcp4 (wlp2s0): state changed bound -> done
Oct  7 16:30:07 popos NetworkManager[743]: <info>  [1696667407.4992] device (wlp2s0): supplicant interface state: completed -> disconnected
Oct  7 16:30:07 popos avahi-daemon[735]: Withdrawing address record for 192.168.3.11 on wlp2s0.
Oct  7 16:30:07 popos NetworkManager[743]: <info>  [1696667407.4993] device (wlp2s0): state change: disconnected -> unmanaged (reason 'sleeping', sys-iface-state: 'managed')
Oct  7 16:30:07 popos avahi-daemon[735]: Leaving mDNS multicast group on interface wlp2s0.IPv4 with address 192.168.3.11.
Oct  7 16:30:07 popos avahi-daemon[735]: Interface wlp2s0.IPv4 no longer relevant for mDNS.
Oct  7 16:30:07 popos nm-dispatcher[6227]: run-parts: failed to stat component /etc/network/if-post-down.d/avahi-daemon: No such file or directory
Oct  7 16:30:07 popos wpa_supplicant[784]: nl80211: deinit ifname=p2p-dev-wlp2s0 disabled_11b_rates=0
Oct  7 16:30:07 popos wpa_supplicant[784]: nl80211: deinit ifname=wlp2s0 disabled_11b_rates=0
Oct  7 16:30:09 popos nmbd[1455]: [2023/10/07 16:30:09.100736,  0] ../../source3/libsmb/nmblib.c:922(send_udp)
Oct  7 16:30:09 popos nmbd[1455]:   Packet send failed to 192.168.3.255(138) ERRNO=网络不可达
Oct  7 16:30:09 popos nmbd[1455]: [2023/10/07 16:30:09.100949,  0] ../../source3/libsmb/nmblib.c:922(send_udp)
Oct  7 16:30:09 popos nmbd[1455]:   Packet send failed to 192.168.3.255(138) ERRNO=网络不可达
Oct  7 16:30:09 popos gnome-shell[2227]: An active wireless connection, in infrastructure mode, involves no access point?
Oct  7 16:30:09 popos gnome-shell[2227]: DING: desktop-icons: Desktop is writable by others - will not allow launching any desktop files#012Gjs-Message: 16:21:56.988: JS LOG: Error connecting to Nautilus#012GNOME nautilus 3.38.1
Oct  7 16:30:09 popos gnome-shell[2227]: ../../../gobject/gsignal.c:2732: instance '0x55669b82d920' has no handler with id '52229'
Oct  7 16:30:10 popos systemd[1]: Reached target Sleep.
Oct  7 16:30:10 popos systemd[1]: Starting NVIDIA system suspend actions...
Oct  7 16:30:10 popos suspend: nvidia-suspend.service
Oct  7 16:30:10 popos logger[6241]: <13>Oct  7 16:30:10 suspend: nvidia-suspend.service
Oct  7 16:30:12 popos /usr/libexec/gdm-x-session[1631]: (**) Option "fd" "56"
Oct  7 16:30:12 popos /usr/libexec/gdm-x-session[1631]: (II) event3  - Power Button: device removed
Oct  7 16:30:12 popos /usr/libexec/gdm-x-session[1631]: (**) Option "fd" "32"
Oct  7 16:30:12 popos /usr/libexec/gdm-x-session[1631]: (II) event11 - Video Bus: device removed
Oct  7 16:30:12 popos /usr/libexec/gdm-x-session[1631]: (**) Option "fd" "58"
Oct  7 16:30:12 popos /usr/libexec/gdm-x-session[1631]: (II) event12 - Video Bus: device removed
Oct  7 16:30:12 popos /usr/libexec/gdm-x-session[1631]: (**) Option "fd" "60"
Oct  7 16:30:12 popos /usr/libexec/gdm-x-session[1631]: (II) event0  - Power Button: device removed
Oct  7 16:30:12 popos /usr/libexec/gdm-x-session[1631]: (**) Option "fd" "53"
Oct  7 16:30:12 popos /usr/libexec/gdm-x-session[1631]: (II) event1  - Sleep Button: device removed
Oct  7 16:30:12 popos /usr/libexec/gdm-x-session[1631]: (**) Option "fd" "57"
Oct  7 16:30:12 popos /usr/libexec/gdm-x-session[1631]: (II) event9  - Logitech M570: device removed
Oct  7 16:30:12 popos /usr/libexec/gdm-x-session[1631]: (**) Option "fd" "59"
Oct  7 16:30:12 popos /usr/libexec/gdm-x-session[1631]: (**) Option "fd" "62"
Oct  7 16:30:12 popos /usr/libexec/gdm-x-session[1631]: (II) event5  - USB Keyboard USB Keyboard: device removed
Oct  7 16:30:12 popos /usr/libexec/gdm-x-session[1631]: (**) Option "fd" "55"
Oct  7 16:30:12 popos /usr/libexec/gdm-x-session[1631]: (**) Option "fd" "54"
Oct  7 16:30:12 popos /usr/libexec/gdm-x-session[1631]: (II) event14 - Chicony USB2.0 Camera: Chicony : device removed
Oct  7 16:30:12 popos /usr/libexec/gdm-x-session[1631]: (**) Option "fd" "82"
Oct  7 16:30:12 popos /usr/libexec/gdm-x-session[1631]: (II) event4  - AT Translated Set 2 keyboard: device removed
Oct  7 16:30:12 popos /usr/libexec/gdm-x-session[1631]: (**) Option "fd" "49"
Oct  7 16:30:12 popos /usr/libexec/gdm-x-session[1631]: (II) event10 - SynPS/2 Synaptics TouchPad: device removed
Oct  7 16:30:12 popos /usr/libexec/gdm-x-session[1631]: (**) Option "fd" "52"
Oct  7 16:30:12 popos /usr/libexec/gdm-x-session[1631]: (**) Option "fd" "59"
Oct  7 16:30:12 popos /usr/libexec/gdm-x-session[1631]: (II) event7  - Logitech M720 Triathlon: device removed
Oct  7 16:30:12 popos /usr/libexec/gdm-x-session[1631]: (**) Option "fd" "55"
Oct  7 16:30:12 popos /usr/libexec/gdm-x-session[1631]: (II) event8  - USB Keyboard USB Keyboard Consumer Control: device removed
Oct  7 16:30:12 popos /usr/libexec/gdm-x-session[1631]: (**) Option "fd" "52"
Oct  7 16:30:12 popos /usr/libexec/gdm-x-session[1631]: (II) event22 - LogiOps Virtual Input: device removed
Oct  7 16:30:12 popos kernel: [  807.064632] rfkill: input handler enabled
Oct  7 16:30:12 popos /usr/libexec/gdm-x-session[1631]: (II) systemd-logind: got pause for 13:73
Oct  7 16:30:12 popos /usr/libexec/gdm-x-session[1631]: (II) systemd-logind: got pause for 13:75
Oct  7 16:30:12 popos /usr/libexec/gdm-x-session[1631]: (II) systemd-logind: got pause for 13:74
Oct  7 16:30:12 popos /usr/libexec/gdm-x-session[1631]: (II) systemd-logind: got pause for 13:86
Oct  7 16:30:12 popos /usr/libexec/gdm-x-session[1631]: (II) systemd-logind: got pause for 13:65
Oct  7 16:30:12 popos /usr/libexec/gdm-x-session[1631]: (II) systemd-logind: got pause for 13:78
Oct  7 16:30:12 popos /usr/libexec/gdm-x-session[1631]: (II) systemd-logind: got pause for 13:72
Oct  7 16:30:12 popos /usr/libexec/gdm-x-session[1631]: (II) systemd-logind: got pause for 13:68
Oct  7 16:30:12 popos /usr/libexec/gdm-x-session[1631]: (II) systemd-logind: got pause for 13:67
Oct  7 16:30:12 popos /usr/libexec/gdm-x-session[1631]: (II) systemd-logind: got pause for 13:76
Oct  7 16:30:12 popos /usr/libexec/gdm-x-session[1631]: (II) systemd-logind: got pause for 13:71
Oct  7 16:30:12 popos /usr/libexec/gdm-x-session[1631]: (II) systemd-logind: got pause for 13:64
Oct  7 16:30:12 popos /usr/libexec/gdm-x-session[1631]: (II) systemd-logind: got pause for 226:0
Oct  7 16:30:12 popos /usr/libexec/gdm-x-session[1631]: (II) systemd-logind: got pause for 13:69
Oct  7 16:30:13 popos systemd[1]: nvidia-suspend.service: Succeeded.
Oct  7 16:30:13 popos systemd[1]: Finished NVIDIA system suspend actions.
Oct  7 16:30:13 popos systemd[1]: Starting Suspend...
Oct  7 16:30:13 popos systemd-sleep[6246]: Suspending system...
Oct  7 16:30:13 popos kernel: [  807.496174] PM: suspend entry (deep)
Oct  7 16:30:13 popos kernel: [  808.101867] Filesystems sync: 0.605 seconds
Oct  7 16:30:22 popos pipewire[1518]: #033[1;33m[W][000000812.530380][main-loop.c:157 pw_main_loop_run()] main-loop 0x55d6e1ab60a0: iterate error -4 (Interrupted system call)#033[0m
Oct  7 16:30:22 popos rtkit-daemon[1000]: The canary thread is apparently starving. Taking action.
Oct  7 16:30:22 popos /usr/libexec/gdm-x-session[1631]: (EE) NVIDIA(GPU-0): WAIT (2, 8, 0x8000, 0x0000e6c4, 0x0000e7b4)
Oct  7 16:30:22 popos pipewire[1536]: #033[1;33m[W][000000812.530972][main-loop.c:157 pw_main_loop_run()] main-loop 0x557a74642e10: iterate error -4 (Interrupted system call)#033[0m
Oct  7 16:30:22 popos rtkit-daemon[1000]: Demoting known real-time threads.
Oct  7 16:30:22 popos rtkit-daemon[1000]: Successfully demoted thread 1524 of process 1518.
Oct  7 16:30:22 popos rtkit-daemon[1000]: Successfully demoted thread 1640 of process 1520.
Oct  7 16:30:22 popos rtkit-daemon[1000]: Successfully demoted thread 1633 of process 1520.
Oct  7 16:30:22 popos rtkit-daemon[1000]: Successfully demoted thread 1520 of process 1520.
Oct  7 16:30:22 popos rtkit-daemon[1000]: Demoted 4 threads.
Oct  7 16:30:22 popos acpid: client 1631[0:0] has disconnected
Oct  7 16:30:22 popos kernel: [  808.218685] Freezing user space processes ... (elapsed 0.002 seconds) done.
Oct  7 16:30:22 popos kernel: [  808.220767] OOM killer disabled.
Oct  7 16:30:22 popos kernel: [  808.220768] Freezing remaining freezable tasks ... (elapsed 0.001 seconds) done.
Oct  7 16:30:22 popos kernel: [  808.222049] printk: Suspending console(s) (use no_console_suspend to debug)
Oct  7 16:30:22 popos kernel: [  808.241327] sd 2:0:0:0: [sda] Synchronizing SCSI cache
Oct  7 16:30:22 popos kernel: [  808.241394] sd 3:0:0:0: [sdb] Synchronizing SCSI cache
Oct  7 16:30:22 popos kernel: [  808.241802] sd 2:0:0:0: [sda] Stopping disk
Oct  7 16:30:22 popos kernel: [  808.242177] sd 3:0:0:0: [sdb] Stopping disk
Oct  7 16:30:22 popos kernel: [  810.105958] ACPI: EC: interrupt blocked
Oct  7 16:30:22 popos kernel: [  810.195070] ACPI: Preparing to enter system sleep state S3
Oct  7 16:30:22 popos kernel: [  810.196745] ACPI: EC: event blocked
Oct  7 16:30:22 popos kernel: [  810.196746] ACPI: EC: EC stopped
Oct  7 16:30:22 popos kernel: [  810.196747] PM: Saving platform NVS memory
Oct  7 16:30:22 popos kernel: [  810.196809] Disabling non-boot CPUs ...
Oct  7 16:30:22 popos kernel: [  810.197295] IRQ 129: no longer affine to CPU1
Oct  7 16:30:22 popos kernel: [  810.198315] smpboot: CPU 1 is now offline
Oct  7 16:30:22 popos kernel: [  810.199505] IRQ 131: no longer affine to CPU2
Oct  7 16:30:22 popos kernel: [  810.200831] smpboot: CPU 2 is now offline
Oct  7 16:30:22 popos kernel: [  810.201851] IRQ 123: no longer affine to CPU3
Oct  7 16:30:22 popos kernel: [  810.201856] IRQ 124: no longer affine to CPU3
Oct  7 16:30:22 popos kernel: [  810.201859] IRQ 125: no longer affine to CPU3
Oct  7 16:30:22 popos kernel: [  810.202878] smpboot: CPU 3 is now offline
Oct  7 16:30:22 popos kernel: [  810.205423] ACPI: Low-level resume complete
Oct  7 16:30:22 popos kernel: [  810.205498] ACPI: EC: EC started
Oct  7 16:30:22 popos kernel: [  810.205498] PM: Restoring platform NVS memory
Oct  7 16:30:22 popos kernel: [  810.207239] Enabling non-boot CPUs ...
Oct  7 16:30:22 popos kernel: [  810.207272] x86: Booting SMP configuration:
Oct  7 16:30:22 popos kernel: [  810.207273] smpboot: Booting Node 0 Processor 1 APIC 0x2
Oct  7 16:30:22 popos kernel: [  810.208803] CPU1 is up
Oct  7 16:30:22 popos kernel: [  810.208825] smpboot: Booting Node 0 Processor 2 APIC 0x1
Oct  7 16:30:22 popos kernel: [  810.209351] CPU2 is up
Oct  7 16:30:22 popos kernel: [  810.209373] smpboot: Booting Node 0 Processor 3 APIC 0x3
Oct  7 16:30:22 popos kernel: [  810.209836] CPU3 is up
Oct  7 16:30:22 popos kernel: [  810.211202] ACPI: Waking up from system sleep state S3
Oct  7 16:30:22 popos kernel: [  810.215508] ACPI: EC: interrupt unblocked
Oct  7 16:30:22 popos kernel: [  811.372023] nvidia 0000:01:00.0: Enabling HDA controller
Oct  7 16:30:22 popos kernel: [  811.373377] ACPI: EC: event unblocked
Oct  7 16:30:22 popos kernel: [  811.374115] sd 2:0:0:0: [sda] Starting disk
Oct  7 16:30:22 popos kernel: [  811.374131] sd 3:0:0:0: [sdb] Starting disk
Oct  7 16:30:22 popos kernel: [  811.376126] iwlwifi 0000:02:00.0: RF_KILL bit toggled to enable radio.
Oct  7 16:30:22 popos kernel: [  811.459907] xhci_hcd 0000:00:14.0: Port resume timed out, port 1-6: 0x400603
Oct  7 16:30:22 popos kernel: [  811.694641] ata4: SATA link up 6.0 Gbps (SStatus 133 SControl 300)
Oct  7 16:30:22 popos kernel: [  811.737008] usb 1-3: reset full-speed USB device number 3 using xhci_hcd
Oct  7 16:30:22 popos kernel: [  811.737444] ata4.00: ACPI cmd ef/10:06:00:00:00:00 (SET FEATURES) succeeded
Oct  7 16:30:22 popos kernel: [  811.737447] ata4.00: ACPI cmd f5/00:00:00:00:00:00 (SECURITY FREEZE LOCK) filtered out
Oct  7 16:30:22 popos kernel: [  811.737449] ata4.00: ACPI cmd b1/c1:00:00:00:00:00 (DEVICE CONFIGURATION OVERLAY) filtered out
Oct  7 16:30:22 popos kernel: [  811.740515] ata4.00: ACPI cmd ef/10:06:00:00:00:00 (SET FEATURES) succeeded
Oct  7 16:30:22 popos kernel: [  811.740522] ata4.00: ACPI cmd f5/00:00:00:00:00:00 (SECURITY FREEZE LOCK) filtered out
Oct  7 16:30:22 popos kernel: [  811.740525] ata4.00: ACPI cmd b1/c1:00:00:00:00:00 (DEVICE CONFIGURATION OVERLAY) filtered out
Oct  7 16:30:22 popos kernel: [  811.742385] ata4.00: configured for UDMA/133
Oct  7 16:30:22 popos kernel: [  811.752466] ahci 0000:00:17.0: port does not support device sleep
Oct  7 16:30:22 popos kernel: [  811.752978] ata4.00: Enabling discard_zeroes_data
Oct  7 16:30:22 popos kernel: [  812.091658] usb 1-9: reset high-speed USB device number 6 using xhci_hcd
Oct  7 16:30:22 popos kernel: [  812.352418] ata3: SATA link up 6.0 Gbps (SStatus 133 SControl 300)
Oct  7 16:30:22 popos kernel: [  812.353911] ata3.00: ACPI cmd ef/10:06:00:00:00:00 (SET FEATURES) succeeded
Oct  7 16:30:22 popos kernel: [  812.353916] ata3.00: ACPI cmd f5/00:00:00:00:00:00 (SECURITY FREEZE LOCK) filtered out
Oct  7 16:30:22 popos kernel: [  812.353918] ata3.00: ACPI cmd b1/c1:00:00:00:00:00 (DEVICE CONFIGURATION OVERLAY) filtered out
Oct  7 16:30:22 popos kernel: [  812.356674] ata3.00: ACPI cmd ef/10:06:00:00:00:00 (SET FEATURES) succeeded
Oct  7 16:30:22 popos kernel: [  812.356678] ata3.00: ACPI cmd f5/00:00:00:00:00:00 (SECURITY FREEZE LOCK) filtered out
Oct  7 16:30:22 popos kernel: [  812.356680] ata3.00: ACPI cmd b1/c1:00:00:00:00:00 (DEVICE CONFIGURATION OVERLAY) filtered out
Oct  7 16:30:22 popos kernel: [  812.357785] ata3.00: configured for UDMA/133
Oct  7 16:30:22 popos kernel: [  812.449202] usb 1-5: reset full-speed USB device number 4 using xhci_hcd
Oct  7 16:30:22 popos kernel: [  812.481853] psmouse serio2: synaptics: queried max coordinates: x [..5674], y [..4764]
Oct  7 16:30:22 popos kernel: [  812.599299] OOM killer enabled.
Oct  7 16:30:22 popos kernel: [  812.599300] Restarting tasks ... 
Oct  7 16:30:22 popos kernel: [  812.613201] mei_hdcp 0000:00:16.0-b638ab7e-94e2-4ea2-a552-d1c54b627f04: bound 0000:00:02.0 (ops i915_hdcp_component_ops [i915])
Oct  7 16:30:22 popos kernel: [  812.615743] done.
Oct  7 16:30:22 popos kernel: [  812.615828] thermal thermal_zone1: failed to read out thermal zone (-61)
Oct  7 16:30:22 popos kernel: [  812.616353] PM: suspend exit
Oct  7 16:30:22 popos systemd[1]: Starting Load/Save RF Kill Switch Status...
Oct  7 16:30:22 popos bluetoothd[736]: Endpoint unregistered: sender=:1.74 path=/MediaEndpoint/A2DPSink/sbc
Oct  7 16:30:22 popos systemd[1]: Started Load/Save RF Kill Switch Status.
Oct  7 16:30:22 popos bluetoothd[736]: Endpoint unregistered: sender=:1.74 path=/MediaEndpoint/A2DPSource/sbc
Oct  7 16:30:22 popos systemd-sleep[6246]: System resumed.
Oct  7 16:30:22 popos systemd[1]: NetworkManager-dispatcher.service: Succeeded.
Oct  7 16:30:22 popos upowerd[1082]: treating change event as add on /sys/devices/pci0000:00/0000:00:14.0/usb1/1-5
Oct  7 16:30:22 popos kernel: [  812.619661] psmouse serio2: synaptics: queried min coordinates: x [1266..], y [1086..]
Oct  7 16:30:22 popos kernel: [  812.685836] Bluetooth: hci0: read Intel version: 370810225019140f00
Oct  7 16:30:22 popos kernel: [  812.685843] Bluetooth: hci0: Intel Bluetooth firmware file: intel/ibt-hw-37.8.10-fw-22.50.19.14.f.bseq
Oct  7 16:30:22 popos upowerd[1082]: treating change event as add on /sys/devices/pci0000:00/0000:00:14.0/usb1/1-5
Oct  7 16:30:22 popos upowerd[1082]: treating change event as add on /sys/devices/pci0000:00/0000:00:14.0/usb1/1-3
Oct  7 16:30:22 popos upowerd[1082]: treating change event as add on /sys/devices/pci0000:00/0000:00:14.0/usb1/1-3
Oct  7 16:30:22 popos systemd[1507]: Stopped target Bluetooth.
Oct  7 16:30:22 popos systemd[1]: Stopped target Bluetooth.
Oct  7 16:30:22 popos systemd[1507]: Reached target Bluetooth.
Oct  7 16:30:22 popos systemd[1]: Reached target Bluetooth.
Oct  7 16:30:22 popos systemd-sleep[6372]: /dev/sda:
Oct  7 16:30:22 popos systemd-sleep[6372]:  setting Advanced Power Management level to 0xfe (254)
Oct  7 16:30:22 popos systemd-sleep[6372]:  APM_level#011= 254
Oct  7 16:30:22 popos kernel: [  813.000155] Bluetooth: hci0: Intel BT fw patch 0x42 completed & activated
Oct  7 16:30:22 popos bluetoothd[736]: Endpoint registered: sender=:1.74 path=/MediaEndpoint/A2DPSink/sbc
Oct  7 16:30:22 popos bluetoothd[736]: Endpoint registered: sender=:1.74 path=/MediaEndpoint/A2DPSource/sbc
Oct  7 16:30:22 popos gdm3: GLib: Source ID 75 was not found when attempting to remove it
Oct  7 16:30:22 popos systemd[1]: systemd-suspend.service: Succeeded.
Oct  7 16:30:22 popos systemd[1]: Finished Suspend.
Oct  7 16:30:22 popos systemd[1]: Stopped target Sleep.
Oct  7 16:30:22 popos systemd[1]: Reached target Suspend.
Oct  7 16:30:22 popos systemd[1]: Starting NVIDIA system resume actions...
Oct  7 16:30:22 popos systemd[1]: Stopped target Suspend.
Oct  7 16:30:22 popos suspend: nvidia-resume.service
Oct  7 16:30:22 popos logger[6439]: <13>Oct  7 16:30:22 suspend: nvidia-resume.service
Oct  7 16:30:22 popos systemd[1]: nvidia-resume.service: Succeeded.
Oct  7 16:30:22 popos systemd[1]: Finished NVIDIA system resume actions.
Oct  7 16:30:23 popos ModemManager[892]: <info>  [sleep-monitor] system is resuming
Oct  7 16:30:23 popos NetworkManager[743]: <info>  [1696667423.1679] manager: sleep: wake requested (sleeping: yes  enabled: yes)
Oct  7 16:30:23 popos NetworkManager[743]: <info>  [1696667423.1680] device (enp3s0f1): state change: unmanaged -> unavailable (reason 'managed', sys-iface-state: 'managed')
Oct  7 16:30:23 popos kernel: [  813.502734] rfkill: input handler disabled
Oct  7 16:30:23 popos kernel: [  813.532281] Generic FE-GE Realtek PHY r8169-301:00: attached PHY driver (mii_bus:phy_addr=r8169-301:00, irq=IGNORE)
Oct  7 16:30:23 popos NetworkManager[743]: <info>  [1696667423.3696] device (wlp2s0): state change: unmanaged -> unavailable (reason 'managed', sys-iface-state: 'managed')
Oct  7 16:30:23 popos kernel: [  813.704387] r8169 0000:03:00.1 enp3s0f1: Link is Down
Oct  7 16:30:23 popos NetworkManager[743]: <info>  [1696667423.4362] device (p2p-dev-wlp2s0): state change: unmanaged -> unavailable (reason 'managed', sys-iface-state: 'managed')
Oct  7 16:30:23 popos NetworkManager[743]: <info>  [1696667423.4366] manager: NetworkManager state is now DISCONNECTED
Oct  7 16:30:23 popos NetworkManager[743]: <info>  [1696667423.4833] device (wlp2s0): supplicant interface state: internal-starting -> disconnected
Oct  7 16:30:23 popos NetworkManager[743]: <info>  [1696667423.4834] device (p2p-dev-wlp2s0): state change: unavailable -> unmanaged (reason 'removed', sys-iface-state: 'removed')
Oct  7 16:30:23 popos NetworkManager[743]: <info>  [1696667423.4839] Wi-Fi P2P device controlled by interface wlp2s0 created
Oct  7 16:30:23 popos NetworkManager[743]: <info>  [1696667423.4841] manager: (p2p-dev-wlp2s0): new 802.11 Wi-Fi P2P device (/org/freedesktop/NetworkManager/Devices/6)
Oct  7 16:30:23 popos NetworkManager[743]: <info>  [1696667423.4843] device (p2p-dev-wlp2s0): state change: unmanaged -> unavailable (reason 'managed', sys-iface-state: 'external')
Oct  7 16:30:23 popos NetworkManager[743]: <info>  [1696667423.4848] device (wlp2s0): state change: unavailable -> disconnected (reason 'supplicant-available', sys-iface-state: 'managed')
Oct  7 16:30:23 popos NetworkManager[743]: <info>  [1696667423.4859] device (p2p-dev-wlp2s0): state change: unavailable -> disconnected (reason 'none', sys-iface-state: 'managed')
Oct  7 16:30:23 popos NetworkManager[743]: <warn>  [1696667423.4973] sup-iface[c2549c2ea2cdb9ac,2,wlp2s0]: call-p2p-cancel: failed with P2P cancel failed
Oct  7 16:30:25 popos ModemManager[892]: <info>  [base-manager] couldn't check support for device '/sys/devices/pci0000:00/0000:00:1c.0/0000:02:00.0': not supported by any plugin
Oct  7 16:30:25 popos ModemManager[892]: <info>  [base-manager] couldn't check support for device '/sys/devices/pci0000:00/0000:00:1c.7/0000:03:00.1': not supported by any plugin
Oct  7 16:30:26 popos /usr/libexec/gdm-x-session[1631]: (EE) NVIDIA(GPU-0): WAIT (1, 8, 0x8000, 0x0000e6c4, 0x0000e7b4)
Oct  7 16:30:26 popos /usr/libexec/gdm-x-session[1631]: (II) systemd-logind: got resume for 13:73
Oct  7 16:30:26 popos /usr/libexec/gdm-x-session[1631]: (II) systemd-logind: got resume for 13:75
Oct  7 16:30:26 popos /usr/libexec/gdm-x-session[1631]: (II) systemd-logind: got resume for 13:74
Oct  7 16:30:26 popos /usr/libexec/gdm-x-session[1631]: (II) systemd-logind: got resume for 13:86
Oct  7 16:30:26 popos /usr/libexec/gdm-x-session[1631]: (II) systemd-logind: got resume for 13:65
Oct  7 16:30:26 popos /usr/libexec/gdm-x-session[1631]: (II) systemd-logind: got resume for 13:78
Oct  7 16:30:26 popos /usr/libexec/gdm-x-session[1631]: (II) systemd-logind: got resume for 13:72
Oct  7 16:30:26 popos /usr/libexec/gdm-x-session[1631]: (II) systemd-logind: got resume for 13:68
Oct  7 16:30:26 popos /usr/libexec/gdm-x-session[1631]: (II) systemd-logind: got resume for 13:67
Oct  7 16:30:26 popos /usr/libexec/gdm-x-session[1631]: (II) systemd-logind: got resume for 13:76
Oct  7 16:30:26 popos /usr/libexec/gdm-x-session[1631]: (II) systemd-logind: got resume for 13:71
Oct  7 16:30:26 popos /usr/libexec/gdm-x-session[1631]: (II) systemd-logind: got resume for 13:64
Oct  7 16:30:26 popos /usr/libexec/gdm-x-session[1631]: (II) systemd-logind: got resume for 226:0
Oct  7 16:30:26 popos /usr/libexec/gdm-x-session[1631]: (II) NVIDIA(0): Setting mode "NULL"
Oct  7 16:30:26 popos acpid: client connected from 1631[0:0]
Oct  7 16:30:26 popos acpid: 1 client rule loaded
Oct  7 16:30:26 popos wpa_supplicant[784]: wlp2s0: CTRL-EVENT-REGDOM-CHANGE init=DRIVER type=COUNTRY alpha2=CN
Oct  7 16:30:26 popos /usr/libexec/gdm-x-session[1631]: (II) event3  - Power Button: is tagged by udev as: Keyboard
Oct  7 16:30:26 popos /usr/libexec/gdm-x-session[1631]: (II) event3  - Power Button: device is a keyboard
Oct  7 16:30:26 popos /usr/libexec/gdm-x-session[1631]: (II) event11 - Video Bus: is tagged by udev as: Keyboard
Oct  7 16:30:26 popos /usr/libexec/gdm-x-session[1631]: (II) event11 - Video Bus: device is a keyboard
Oct  7 16:30:26 popos /usr/libexec/gdm-x-session[1631]: (II) event12 - Video Bus: is tagged by udev as: Keyboard
Oct  7 16:30:26 popos /usr/libexec/gdm-x-session[1631]: (II) event12 - Video Bus: device is a keyboard
Oct  7 16:30:26 popos /usr/libexec/gdm-x-session[1631]: (II) event0  - Power Button: is tagged by udev as: Keyboard
Oct  7 16:30:26 popos /usr/libexec/gdm-x-session[1631]: (II) event0  - Power Button: device is a keyboard
Oct  7 16:30:26 popos /usr/libexec/gdm-x-session[1631]: (II) event1  - Sleep Button: is tagged by udev as: Keyboard
Oct  7 16:30:26 popos /usr/libexec/gdm-x-session[1631]: (II) event1  - Sleep Button: device is a keyboard
Oct  7 16:30:26 popos /usr/libexec/gdm-x-session[1631]: (II) event9  - Logitech M570: is tagged by udev as: Mouse Trackball
Oct  7 16:30:26 popos /usr/libexec/gdm-x-session[1631]: (II) event9  - Logitech M570: device set to 540 DPI
Oct  7 16:30:26 popos /usr/libexec/gdm-x-session[1631]: (II) event9  - Logitech M570: device is a pointer
Oct  7 16:30:26 popos /usr/libexec/gdm-x-session[1631]: (II) event7  - Logitech M720 Triathlon: is tagged by udev as: Keyboard Mouse
Oct  7 16:30:26 popos /usr/libexec/gdm-x-session[1631]: (II) event7  - Logitech M720 Triathlon: device is a pointer
Oct  7 16:30:26 popos /usr/libexec/gdm-x-session[1631]: (II) event7  - Logitech M720 Triathlon: device is a keyboard
Oct  7 16:30:26 popos /usr/libexec/gdm-x-session[1631]: (II) event8  - USB Keyboard USB Keyboard Consumer Control: is tagged by udev as: Keyboard
Oct  7 16:30:26 popos /usr/libexec/gdm-x-session[1631]: (II) event8  - USB Keyboard USB Keyboard Consumer Control: device is a keyboard
Oct  7 16:30:26 popos /usr/libexec/gdm-x-session[1631]: (II) event14 - Chicony USB2.0 Camera: Chicony : is tagged by udev as: Keyboard
Oct  7 16:30:26 popos /usr/libexec/gdm-x-session[1631]: (II) event14 - Chicony USB2.0 Camera: Chicony : device is a keyboard
Oct  7 16:30:26 popos /usr/libexec/gdm-x-session[1631]: (II) event4  - AT Translated Set 2 keyboard: is tagged by udev as: Keyboard
Oct  7 16:30:26 popos /usr/libexec/gdm-x-session[1631]: (II) event4  - AT Translated Set 2 keyboard: device is a keyboard
Oct  7 16:30:26 popos /usr/libexec/gdm-x-session[1631]: (II) event10 - SynPS/2 Synaptics TouchPad: is tagged by udev as: Touchpad
Oct  7 16:30:26 popos /usr/libexec/gdm-x-session[1631]: (II) event10 - SynPS/2 Synaptics TouchPad: device is a touchpad
Oct  7 16:30:26 popos /usr/libexec/gdm-x-session[1631]: (II) event22 - LogiOps Virtual Input: is tagged by udev as: Keyboard Mouse
Oct  7 16:30:26 popos /usr/libexec/gdm-x-session[1631]: (II) event22 - LogiOps Virtual Input: device is a pointer
Oct  7 16:30:26 popos /usr/libexec/gdm-x-session[1631]: (II) event22 - LogiOps Virtual Input: device is a keyboard
Oct  7 16:30:26 popos /usr/libexec/gdm-x-session[1631]: (II) systemd-logind: got resume for 13:69
Oct  7 16:30:26 popos /usr/libexec/gdm-x-session[1631]: (II) event5  - USB Keyboard USB Keyboard: is tagged by udev as: Keyboard
Oct  7 16:30:26 popos /usr/libexec/gdm-x-session[1631]: (II) event5  - USB Keyboard USB Keyboard: device is a keyboard
Oct  7 16:30:27 popos systemd[1]: systemd-rfkill.service: Succeeded.
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.6882] policy: auto-activating connection '2817' (a3592c45-bd84-44e9-8b20-59f6df1381fa)
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.6887] device (wlp2s0): Activation: starting connection '2817' (a3592c45-bd84-44e9-8b20-59f6df1381fa)
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.6888] device (wlp2s0): state change: disconnected -> prepare (reason 'none', sys-iface-state: 'managed')
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.6892] manager: NetworkManager state is now CONNECTING
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.6896] device (wlp2s0): state change: prepare -> config (reason 'none', sys-iface-state: 'managed')
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.6901] device (wlp2s0): Activation: (wifi) access point '2817' has security, but secrets are required.
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.6901] device (wlp2s0): state change: config -> need-auth (reason 'none', sys-iface-state: 'managed')
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.6920] device (wlp2s0): state change: need-auth -> prepare (reason 'none', sys-iface-state: 'managed')
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.6924] device (wlp2s0): state change: prepare -> config (reason 'none', sys-iface-state: 'managed')
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.6927] device (wlp2s0): Activation: (wifi) connection '2817' has security, and secrets exist.  No new secrets needed.
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.6929] Config: added 'ssid' value '2817'
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.6929] Config: added 'scan_ssid' value '1'
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.6929] Config: added 'bgscan' value 'simple:30:-70:86400'
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.6929] Config: added 'key_mgmt' value 'WPA-PSK WPA-PSK-SHA256 FT-PSK'
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.6929] Config: added 'auth_alg' value 'OPEN'
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.6929] Config: added 'psk' value '<hidden>'
Oct  7 16:30:27 popos wpa_supplicant[784]: wlp2s0: SME: Trying to authenticate with c2:24:80:7f:9d:a6 (SSID='2817' freq=5805 MHz)
Oct  7 16:30:27 popos kernel: [  818.047585] wlp2s0: authenticate with c2:24:80:7f:9d:a6
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.7166] device (wlp2s0): supplicant interface state: disconnected -> authenticating
Oct  7 16:30:27 popos kernel: [  818.050001] wlp2s0: send auth to c2:24:80:7f:9d:a6 (try 1/3)
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.7166] device (p2p-dev-wlp2s0): supplicant management interface state: disconnected -> authenticating
Oct  7 16:30:27 popos wpa_supplicant[784]: wlp2s0: Trying to associate with c2:24:80:7f:9d:a6 (SSID='2817' freq=5805 MHz)
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.7283] device (wlp2s0): supplicant interface state: authenticating -> associating
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.7283] device (p2p-dev-wlp2s0): supplicant management interface state: authenticating -> associating
Oct  7 16:30:27 popos kernel: [  818.064191] wlp2s0: authenticated
Oct  7 16:30:27 popos kernel: [  818.064965] wlp2s0: associate with c2:24:80:7f:9d:a6 (try 1/3)
Oct  7 16:30:27 popos kernel: [  818.067527] wlp2s0: RX AssocResp from c2:24:80:7f:9d:a6 (capab=0x1031 status=0 aid=3)
Oct  7 16:30:27 popos kernel: [  818.068824] wlp2s0: associated
Oct  7 16:30:27 popos wpa_supplicant[784]: wlp2s0: Associated with c2:24:80:7f:9d:a6
Oct  7 16:30:27 popos wpa_supplicant[784]: wlp2s0: CTRL-EVENT-SUBNET-STATUS-UPDATE status=0
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.7381] device (wlp2s0): supplicant interface state: associating -> associated
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.7382] device (p2p-dev-wlp2s0): supplicant management interface state: associating -> associated
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.7600] device (wlp2s0): supplicant interface state: associated -> 4way_handshake
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.7600] device (p2p-dev-wlp2s0): supplicant management interface state: associated -> 4way_handshake
Oct  7 16:30:27 popos wpa_supplicant[784]: wlp2s0: WPA: Key negotiation completed with c2:24:80:7f:9d:a6 [PTK=CCMP GTK=CCMP]
Oct  7 16:30:27 popos wpa_supplicant[784]: wlp2s0: CTRL-EVENT-CONNECTED - Connection to c2:24:80:7f:9d:a6 completed [id=0 id_str=]
Oct  7 16:30:27 popos wpa_supplicant[784]: wlp2s0: CTRL-EVENT-SIGNAL-CHANGE above=0 signal=-51 noise=9999 txrate=6000
Oct  7 16:30:27 popos kernel: [  818.102631] IPv6: ADDRCONF(NETDEV_CHANGE): wlp2s0: link becomes ready
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.7715] device (wlp2s0): supplicant interface state: 4way_handshake -> completed
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.7715] device (wlp2s0): Activation: (wifi) Stage 2 of 5 (Device Configure) successful. Connected to wireless network "2817"
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.7715] device (p2p-dev-wlp2s0): supplicant management interface state: 4way_handshake -> completed
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.7717] device (wlp2s0): state change: config -> ip-config (reason 'none', sys-iface-state: 'managed')
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.7720] dhcp4 (wlp2s0): activation: beginning transaction (timeout in 45 seconds)
Oct  7 16:30:27 popos avahi-daemon[735]: Joining mDNS multicast group on interface wlp2s0.IPv6 with address fe80::9139:382a:f7a5:ed88.
Oct  7 16:30:27 popos avahi-daemon[735]: New relevant interface wlp2s0.IPv6 for mDNS.
Oct  7 16:30:27 popos avahi-daemon[735]: Registering new address record for fe80::9139:382a:f7a5:ed88 on wlp2s0.*.
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.7918] dhcp4 (wlp2s0): option dhcp_lease_time      => '86400'
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.7918] dhcp4 (wlp2s0): option domain_name_servers  => '192.168.3.1'
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.7918] dhcp4 (wlp2s0): option expiry               => '1696753827'
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.7918] dhcp4 (wlp2s0): option ip_address           => '192.168.3.11'
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.7918] dhcp4 (wlp2s0): option requested_broadcast_address => '1'
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.7918] dhcp4 (wlp2s0): option requested_domain_name => '1'
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.7918] dhcp4 (wlp2s0): option requested_domain_name_servers => '1'
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.7918] dhcp4 (wlp2s0): option requested_domain_search => '1'
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.7919] dhcp4 (wlp2s0): option requested_host_name  => '1'
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.7919] dhcp4 (wlp2s0): option requested_interface_mtu => '1'
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.7919] dhcp4 (wlp2s0): option requested_ms_classless_static_routes => '1'
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.7919] dhcp4 (wlp2s0): option requested_nis_domain => '1'
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.7919] dhcp4 (wlp2s0): option requested_nis_servers => '1'
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.7919] dhcp4 (wlp2s0): option requested_ntp_servers => '1'
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.7919] dhcp4 (wlp2s0): option requested_rfc3442_classless_static_routes => '1'
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.7919] dhcp4 (wlp2s0): option requested_root_path  => '1'
Oct  7 16:30:27 popos avahi-daemon[735]: Joining mDNS multicast group on interface wlp2s0.IPv4 with address 192.168.3.11.
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.7919] dhcp4 (wlp2s0): option requested_routers    => '1'
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.7919] dhcp4 (wlp2s0): option requested_static_routes => '1'
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.7919] dhcp4 (wlp2s0): option requested_subnet_mask => '1'
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.7919] dhcp4 (wlp2s0): option requested_time_offset => '1'
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.7920] dhcp4 (wlp2s0): option requested_wpad       => '1'
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.7920] dhcp4 (wlp2s0): option routers              => '192.168.3.1'
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.7920] dhcp4 (wlp2s0): option subnet_mask          => '255.255.255.0'
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.7920] dhcp4 (wlp2s0): state changed unknown -> bound
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.7946] device (wlp2s0): state change: ip-config -> ip-check (reason 'none', sys-iface-state: 'managed')
Oct  7 16:30:27 popos wpa_supplicant[784]: wlp2s0: CTRL-EVENT-SIGNAL-CHANGE above=1 signal=-51 noise=9999 txrate=433300
Oct  7 16:30:27 popos avahi-daemon[735]: New relevant interface wlp2s0.IPv4 for mDNS.
Oct  7 16:30:27 popos avahi-daemon[735]: Registering new address record for 192.168.3.11 on wlp2s0.IPv4.
Oct  7 16:30:27 popos dbus-daemon[740]: [system] Activating via systemd: service name='org.freedesktop.nm_dispatcher' unit='dbus-org.freedesktop.nm-dispatcher.service' requested by ':1.13' (uid=0 pid=743 comm="/usr/sbin/NetworkManager --no-daemon " label="unconfined")
Oct  7 16:30:27 popos systemd[1]: Starting Network Manager Script Dispatcher Service...
Oct  7 16:30:27 popos dbus-daemon[740]: [system] Successfully activated service 'org.freedesktop.nm_dispatcher'
Oct  7 16:30:27 popos systemd[1]: Started Network Manager Script Dispatcher Service.
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.8060] device (wlp2s0): state change: ip-check -> secondaries (reason 'none', sys-iface-state: 'managed')
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.8063] device (wlp2s0): state change: secondaries -> activated (reason 'none', sys-iface-state: 'managed')
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.8067] manager: NetworkManager state is now CONNECTED_LOCAL
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.8087] manager: NetworkManager state is now CONNECTED_SITE
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.8089] policy: set '2817' (wlp2s0) as default for IPv4 routing and DNS
Oct  7 16:30:27 popos NetworkManager[743]: <info>  [1696667427.8093] device (wlp2s0): Activation: successful, device activated.
Oct  7 16:30:27 popos systemd-resolved[723]: Using degraded feature set UDP instead of UDP+EDNS0 for DNS server 192.168.3.1.
Oct  7 16:30:27 popos nm-dispatcher[6493]: /etc/network/if-up.d/resolved: 12: mystatedir: not found
Oct  7 16:30:27 popos nm-dispatcher[6513]: /etc/network/if-up.d/resolved: 12: mystatedir: not found
Oct  7 16:30:27 popos gnome-shell[2227]: Removing a network device that was not added
Oct  7 16:30:29 popos NetworkManager[743]: <info>  [1696667429.1349] manager: NetworkManager state is now CONNECTED_GLOBAL
Oct  7 16:30:29 popos dbus-daemon[740]: [system] Activating via systemd: service name='net.reactivated.Fprint' unit='fprintd.service' requested by ':1.85' (uid=1000 pid=2227 comm="/usr/bin/gnome-shell " label="unconfined")
Oct  7 16:30:29 popos systemd[1]: Starting Fingerprint Authentication Daemon...
Oct  7 16:30:29 popos dbus-daemon[740]: [system] Successfully activated service 'net.reactivated.Fprint'
Oct  7 16:30:29 popos systemd[1]: Started Fingerprint Authentication Daemon.
Oct  7 16:30:30 popos /usr/libexec/gdm-x-session[1631]: (EE) event5  - USB Keyboard USB Keyboard: client bug: event processing lagging behind by 28ms, your system is too slow
Oct  7 16:30:32 popos system76-power[770]: [INFO] DBUS Received GetSwitchable() method
Oct  7 16:30:32 popos system76-power[770]: [INFO] DBUS Received GetGraphics() method
Oct  7 16:30:32 popos system76-power[770]: [INFO] DBUS Received GetProfile() method
Oct  7 16:30:32 popos gnome-shell[2227]: gnome-shell-extension-system76-power: power profile was set: 'Balanced'
Oct  7 16:30:32 popos gnome-shell[2227]: Device -> 
Oct  7 16:30:32 popos dbus-daemon[1526]: [session uid=1000 pid=1526] Activating service name='org.gnome.Nautilus' requested by ':1.152' (uid=1000 pid=6545 comm="gjs /usr/share/gnome-shell/extensions/ding@rasters" label="unconfined")
Oct  7 16:30:32 popos dbus-daemon[1526]: [session uid=1000 pid=1526] Activating service name='org.freedesktop.FileManager1' requested by ':1.152' (uid=1000 pid=6545 comm="gjs /usr/share/gnome-shell/extensions/ding@rasters" label="unconfined")
Oct  7 16:30:32 popos dbus-daemon[1526]: [session uid=1000 pid=1526] Activating service name='org.gnome.ArchiveManager1' requested by ':1.152' (uid=1000 pid=6545 comm="gjs /usr/share/gnome-shell/extensions/ding@rasters" label="unconfined")
Oct  7 16:30:32 popos NetworkManager[743]: <info>  [1696667432.9738] agent-manager: agent[a218019edb5c802d,:1.85/org.gnome.Shell.NetworkAgent/1000]: agent registered
Oct  7 16:30:33 popos dbus-daemon[1526]: [session uid=1000 pid=1526] Successfully activated service 'org.gnome.Nautilus'
Oct  7 16:30:33 popos gnome-shell[2227]: JS ERROR: GLib.MarkupError: 1:424: Only direction 'out' is allowed for <arg> elements embedded in <signal>#012_newNodeInfo@resource:///org/gnome/gjs/modules/core/overrides/Gio.js:276:16#012_wrapFunction/klass[method]@resource:///org/gnome/gjs/modules/core/overrides/Gio.js:309:25#012introspectBusObject/<@/usr/share/gnome-shell/extensions/ubuntu-appindicators@ubuntu.com/util.js:173:50
Oct  7 16:30:33 popos org.gnome.Nautilus[6558]: Failed to register: Unable to acquire bus name 'org.gnome.Nautilus'
Oct  7 16:30:33 popos dbus-daemon[1526]: [session uid=1000 pid=1526] Successfully activated service 'org.gnome.ArchiveManager1'
Oct  7 16:30:33 popos dbus-daemon[1526]: [session uid=1000 pid=1526] Successfully activated service 'org.freedesktop.FileManager1'
Oct  7 16:30:33 popos gnome-shell[2227]: Window manager warning: Overwriting existing binding of keysym 71 with keysym 71 (keycode 18).
Oct  7 16:30:39 popos systemd[1]: NetworkManager-dispatcher.service: Succeeded.
Oct  7 16:30:41 popos systemd[1507]: Started VTE child process 6644 launched by gnome-terminal-server process 4222.
Oct  7 16:30:41 popos gnome-shell[2227]: Window manager warning: Overwriting existing binding of keysym 71 with keysym 71 (keycode 18).
Oct  7 16:30:45 popos dbus-daemon[1526]: [session uid=1000 pid=1526] Activating via systemd: service name='org.freedesktop.Tracker1' unit='tracker-store.service' requested by ':1.2' (uid=1000 pid=1523 comm="/usr/libexec/tracker-miner-fs " label="unconfined")
Oct  7 16:30:45 popos systemd[1507]: Starting Tracker metadata database store and lookup manager...
Oct  7 16:30:45 popos dbus-daemon[1526]: [session uid=1000 pid=1526] Successfully activated service 'org.freedesktop.Tracker1'
Oct  7 16:30:45 popos systemd[1507]: Started Tracker metadata database store and lookup manager.
Oct  7 16:30:59 popos systemd[1]: fprintd.service: Succeeded.
Oct  7 16:31:15 popos tracker-store[6703]: OK
```

# dmesg

```messages
[Sat Oct  7 16:30:09 2023] wlp2s0: deauthenticating from c2:24:80:7f:9d:a6 by local choice (Reason: 3=DEAUTH_LEAVING)
[Sat Oct  7 16:30:15 2023] rfkill: input handler enabled
[Sat Oct  7 16:30:15 2023] PM: suspend entry (deep)
[Sat Oct  7 16:30:16 2023] Filesystems sync: 0.605 seconds
[Sat Oct  7 16:30:16 2023] Freezing user space processes ... (elapsed 0.002 seconds) done.
[Sat Oct  7 16:30:16 2023] OOM killer disabled.
[Sat Oct  7 16:30:16 2023] Freezing remaining freezable tasks ... (elapsed 0.001 seconds) done.
[Sat Oct  7 16:30:16 2023] printk: Suspending console(s) (use no_console_suspend to debug)
[Sat Oct  7 16:30:16 2023] sd 2:0:0:0: [sda] Synchronizing SCSI cache
[Sat Oct  7 16:30:16 2023] sd 3:0:0:0: [sdb] Synchronizing SCSI cache
[Sat Oct  7 16:30:16 2023] sd 2:0:0:0: [sda] Stopping disk
[Sat Oct  7 16:30:16 2023] sd 3:0:0:0: [sdb] Stopping disk
[Sat Oct  7 16:30:18 2023] ACPI: EC: interrupt blocked
[Sat Oct  7 16:30:18 2023] ACPI: Preparing to enter system sleep state S3
[Sat Oct  7 16:30:18 2023] ACPI: EC: event blocked
[Sat Oct  7 16:30:18 2023] ACPI: EC: EC stopped
[Sat Oct  7 16:30:18 2023] PM: Saving platform NVS memory
[Sat Oct  7 16:30:18 2023] Disabling non-boot CPUs ...
[Sat Oct  7 16:30:18 2023] IRQ 129: no longer affine to CPU1
[Sat Oct  7 16:30:18 2023] smpboot: CPU 1 is now offline
[Sat Oct  7 16:30:18 2023] IRQ 131: no longer affine to CPU2
[Sat Oct  7 16:30:18 2023] smpboot: CPU 2 is now offline
[Sat Oct  7 16:30:18 2023] IRQ 123: no longer affine to CPU3
[Sat Oct  7 16:30:18 2023] IRQ 124: no longer affine to CPU3
[Sat Oct  7 16:30:18 2023] IRQ 125: no longer affine to CPU3
[Sat Oct  7 16:30:18 2023] smpboot: CPU 3 is now offline
[Sat Oct  7 16:30:18 2023] ACPI: Low-level resume complete
[Sat Oct  7 16:30:18 2023] ACPI: EC: EC started
[Sat Oct  7 16:30:18 2023] PM: Restoring platform NVS memory
[Sat Oct  7 16:30:18 2023] Enabling non-boot CPUs ...
[Sat Oct  7 16:30:18 2023] x86: Booting SMP configuration:
[Sat Oct  7 16:30:18 2023] smpboot: Booting Node 0 Processor 1 APIC 0x2
[Sat Oct  7 16:30:18 2023] CPU1 is up
[Sat Oct  7 16:30:18 2023] smpboot: Booting Node 0 Processor 2 APIC 0x1
[Sat Oct  7 16:30:18 2023] CPU2 is up
[Sat Oct  7 16:30:18 2023] smpboot: Booting Node 0 Processor 3 APIC 0x3
[Sat Oct  7 16:30:18 2023] CPU3 is up
[Sat Oct  7 16:30:18 2023] ACPI: Waking up from system sleep state S3
[Sat Oct  7 16:30:18 2023] ACPI: EC: interrupt unblocked
[Sat Oct  7 16:30:19 2023] nvidia 0000:01:00.0: Enabling HDA controller
[Sat Oct  7 16:30:19 2023] ACPI: EC: event unblocked
[Sat Oct  7 16:30:19 2023] sd 2:0:0:0: [sda] Starting disk
[Sat Oct  7 16:30:19 2023] sd 3:0:0:0: [sdb] Starting disk
[Sat Oct  7 16:30:19 2023] iwlwifi 0000:02:00.0: RF_KILL bit toggled to enable radio.
[Sat Oct  7 16:30:19 2023] xhci_hcd 0000:00:14.0: Port resume timed out, port 1-6: 0x400603
[Sat Oct  7 16:30:19 2023] ata4: SATA link up 6.0 Gbps (SStatus 133 SControl 300)
[Sat Oct  7 16:30:19 2023] usb 1-3: reset full-speed USB device number 3 using xhci_hcd
[Sat Oct  7 16:30:19 2023] ata4.00: ACPI cmd ef/10:06:00:00:00:00 (SET FEATURES) succeeded
[Sat Oct  7 16:30:19 2023] ata4.00: ACPI cmd f5/00:00:00:00:00:00 (SECURITY FREEZE LOCK) filtered out
[Sat Oct  7 16:30:19 2023] ata4.00: ACPI cmd b1/c1:00:00:00:00:00 (DEVICE CONFIGURATION OVERLAY) filtered out
[Sat Oct  7 16:30:19 2023] ata4.00: ACPI cmd ef/10:06:00:00:00:00 (SET FEATURES) succeeded
[Sat Oct  7 16:30:19 2023] ata4.00: ACPI cmd f5/00:00:00:00:00:00 (SECURITY FREEZE LOCK) filtered out
[Sat Oct  7 16:30:19 2023] ata4.00: ACPI cmd b1/c1:00:00:00:00:00 (DEVICE CONFIGURATION OVERLAY) filtered out
[Sat Oct  7 16:30:19 2023] ata4.00: configured for UDMA/133
[Sat Oct  7 16:30:19 2023] ahci 0000:00:17.0: port does not support device sleep
[Sat Oct  7 16:30:19 2023] ata4.00: Enabling discard_zeroes_data
[Sat Oct  7 16:30:20 2023] usb 1-9: reset high-speed USB device number 6 using xhci_hcd
[Sat Oct  7 16:30:20 2023] ata3: SATA link up 6.0 Gbps (SStatus 133 SControl 300)
[Sat Oct  7 16:30:20 2023] ata3.00: ACPI cmd ef/10:06:00:00:00:00 (SET FEATURES) succeeded
[Sat Oct  7 16:30:20 2023] ata3.00: ACPI cmd f5/00:00:00:00:00:00 (SECURITY FREEZE LOCK) filtered out
[Sat Oct  7 16:30:20 2023] ata3.00: ACPI cmd b1/c1:00:00:00:00:00 (DEVICE CONFIGURATION OVERLAY) filtered out
[Sat Oct  7 16:30:20 2023] ata3.00: ACPI cmd ef/10:06:00:00:00:00 (SET FEATURES) succeeded
[Sat Oct  7 16:30:20 2023] ata3.00: ACPI cmd f5/00:00:00:00:00:00 (SECURITY FREEZE LOCK) filtered out
[Sat Oct  7 16:30:20 2023] ata3.00: ACPI cmd b1/c1:00:00:00:00:00 (DEVICE CONFIGURATION OVERLAY) filtered out
[Sat Oct  7 16:30:20 2023] ata3.00: configured for UDMA/133
[Sat Oct  7 16:30:20 2023] usb 1-5: reset full-speed USB device number 4 using xhci_hcd
[Sat Oct  7 16:30:20 2023] psmouse serio2: synaptics: queried max coordinates: x [..5674], y [..4764]
[Sat Oct  7 16:30:20 2023] OOM killer enabled.
[Sat Oct  7 16:30:20 2023] Restarting tasks ... 
[Sat Oct  7 16:30:20 2023] mei_hdcp 0000:00:16.0-b638ab7e-94e2-4ea2-a552-d1c54b627f04: bound 0000:00:02.0 (ops i915_hdcp_component_ops [i915])
[Sat Oct  7 16:30:20 2023] done.
[Sat Oct  7 16:30:20 2023] thermal thermal_zone1: failed to read out thermal zone (-61)
[Sat Oct  7 16:30:20 2023] PM: suspend exit
[Sat Oct  7 16:30:20 2023] psmouse serio2: synaptics: queried min coordinates: x [1266..], y [1086..]
[Sat Oct  7 16:30:20 2023] Bluetooth: hci0: read Intel version: 370810225019140f00
[Sat Oct  7 16:30:20 2023] Bluetooth: hci0: Intel Bluetooth firmware file: intel/ibt-hw-37.8.10-fw-22.50.19.14.f.bseq
[Sat Oct  7 16:30:21 2023] Bluetooth: hci0: Intel BT fw patch 0x42 completed & activated
[Sat Oct  7 16:30:21 2023] rfkill: input handler disabled
[Sat Oct  7 16:30:21 2023] Generic FE-GE Realtek PHY r8169-301:00: attached PHY driver (mii_bus:phy_addr=r8169-301:00, irq=IGNORE)
[Sat Oct  7 16:30:21 2023] r8169 0000:03:00.1 enp3s0f1: Link is Down
[Sat Oct  7 16:30:26 2023] wlp2s0: authenticate with c2:24:80:7f:9d:a6
[Sat Oct  7 16:30:26 2023] wlp2s0: send auth to c2:24:80:7f:9d:a6 (try 1/3)
[Sat Oct  7 16:30:26 2023] wlp2s0: authenticated
[Sat Oct  7 16:30:26 2023] wlp2s0: associate with c2:24:80:7f:9d:a6 (try 1/3)
[Sat Oct  7 16:30:26 2023] wlp2s0: RX AssocResp from c2:24:80:7f:9d:a6 (capab=0x1031 status=0 aid=3)
[Sat Oct  7 16:30:26 2023] wlp2s0: associated
[Sat Oct  7 16:30:26 2023] IPv6: ADDRCONF(NETDEV_CHANGE): wlp2s0: link becomes ready

```

**Comment by theocodes** [comment]
I've been experiencing the same on Debian 12 (Gnome)

```
$ wezterm --version
wezterm 20231003-095118-616e600c
```

**Comment by TomaszJudym** [comment]
Same on Endavour
```
$ wezterm --version
wezterm 20230712-072601-f4abf8fd
$ cat /etc/os-release                            
NAME="EndeavourOS"
PRETTY_NAME="EndeavourOS"
ID="endeavouros"
ID_LIKE="arch"
BUILD_ID="2023.05.28"

```

**Comment by wez** [comment]
This is a problem with the nvidia drivers, there is nothing I can do about it
